charts/kubernikus-system/nginx-ingress/values.yaml

```diff
@@ -0,0 +1,245 @@
+## nginx configuration
+## Ref: https://github.com/kubernetes/ingress/blob/master/controllers/nginx/configuration.md
+##
+controller:
+  name: controller
+  image:
+    repository: gcr.io/google_containers/nginx-ingress-controller
+    tag: "0.9.0-beta.14"
```

Comment:
Any reason we are not using latest (beta.17)?

Reply:
Right. I'll test in staging first and update this PR afterwards